Directory.Build.targets

```diff
@@ -0,0 +1,11 @@
+<Project>
+
+  <!-- We can control common package version (or all of them) in this file. Individual projects have to opt-in to get them
```

Comment:
Could you please add a tiny disclaimer that this file is imported _after_ projects? So it's clear why we have both of them.

Reply:
I moved the `ReferenceAssemblies` to the `props` file as you suggested below. So now there's no need for the `targets` file anymore.